src/Identity/UI/src/Areas/Identity/Pages/V4/Account/Register.cshtml.cs

```diff
@@ -138,7 +138,7 @@ public override async Task<IActionResult> OnPostAsync(string returnUrl = null)
 
                 if (result.Succeeded)
                 {
-                    _logger.LogInformation("User created a new account with password.");
+                    _logger.LogInformation(LoggerEventIds.UserCreated, "User created a new account with password.");
```

Comment:
@blowdart any concerns with logging using an event id in the library and scaffolding still logging like before?

Reply:
Yes actually. If once you scaffold your log event ids change that's a bad thing. They need to match.

Reply:
So are you saying you prefer the UI just doesn't specify event ids?  It seems bad to scaffold out a new file just to add event ids

Reply:
@Kahbazi I chatted with @blowdart offline and looks like this is pretty much ready to go except for one minor tweak, can you make the LoggerEventIds class public that the Identity UI code uses, since we want to be able to reference that in the scaffolded code.  When we port these changes to the scaffolded pages